packages/create-redwood-app/template/scripts/prerender.ts

```diff
@@ -0,0 +1,18 @@
+// import { db } from '$api/src/lib/db'
+
+import type { AvailableRoutes } from '@redwoodjs/router'
+
+// TODO: This isn't picking up the generated names in .redwood/. How can I make
```

Comment:
@dac09 Do you know how to make this work?
I tried adding `"include": ["../.redwood/types/includes/web-*"]` to tsconfig.json, but then the `$api/...` imports stopped working.

Reply:
Not using this file anymore